### ISSUE TITLE
Email recipients off-by-one

### ISSUE THREAD

**Issue by vydrazde** [cmu-sei/GHOSTS#339]
When creating a list of email recipients, if it is non-empty, there is one more recipient than expected for each domain and outside emails.

Likely since https://github.com/cmu-sei/GHOSTS/commit/c2990327f4f2612f1da1586bf341494d4eea3e5c

Log snippet
```
Trace|Ghosts.Client.Infrastructure.Email.EmailConfiguration.ParseEmail|Parsing email - raw Random min 1 max 3
Trace|Ghosts.Client.Infrastructure.Email.EmailConfiguration.ParseEmail|Building domain email list: 14...
Trace|Ghosts.Client.Infrastructure.Email.EmailConfiguration.ParseEmail|Building outside email list: 190...
Trace|Ghosts.Client.Infrastructure.Email.EmailConfiguration.ParseEmail|List generated: 5...
```

**Comment by sei-dupdyke** [comment]
Thanks for submitting — Fixed in 8.0.20, now the default download.

**Comment by vydrazde** [comment]
Thanks for the fix :)
